SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -110,17 +110,14 @@ + (instancetype)outgoingMessageInThread:(nullable TSThread *)thread
         [attachmentIds addObject:attachmentId];
     }
 
-    TSGroupMetaMessage groupMetaMessage =
-        [thread isKindOfClass:TSGroupThread.class] ? TSGroupMessageDeliver : TSGroupMessageNone;
-
```

Comment:
Setting "Deliver" was inadvertently removed here: https://github.com/signalapp/Signal-iOS/commit/41cfdb1536c06d8edaea633c571f7e3e38e36a36

My first attempt to fix it, here, was too narrow. I've moved it down a level.